Server/sprawl_calculator/calculator/urls.py

```diff
@@ -0,0 +1,8 @@
+from django.urls import path
+from . import views
+
+#Url for endpoint1 and endpoint2
+urlpatterns = [
+    path('v1/wup_index>',endpoint1, name='endpoint1'),
```

Comment:
Lets rename the second parameter in the path currently `endpoint` -> `wup_index_generator`